### ISSUE TITLE
Wrong name appears in lower parent cells during expand animation

### ISSUE THREAD

**Issue by hsnetzer** [Vkt0r/AccordionSwift#47]
![IMG_2252 TRIM](https://user-images.githubusercontent.com/18687397/55290223-869ec100-539e-11e9-8c1e-87d6920d73f8.gif)

**Comment by DewaldDeJager** [comment]
@hsnetzer I have not seen this issue. Could you perhaps share the code for reproducing this? It might be that it is in the client-side code for setting up the data source rather than in the framework itself. If it is in the framework, we will fix it but need a bit more info to reproduce it.

**Comment by hsnetzer** [comment]
I'm In the middle of refactoring my app, I want to help reproduce this but I might have to wait a few weeks. The problem could be that I'm scrolling the table view, like this: 

`let didSelectParentCell = { (tableView: UITableView, indexPath: IndexPath, item: ParentCellModel?) -> Void in
            tableView.scrollToRow(at: indexPath, at: UITableView.ScrollPosition.top, animated: true)
        }`

**Comment by hsnetzer** [comment]
Yeah, the scrollToRow causes funky animations. I demonstrate it in the fork here: https://github.com/hsnetzer/AccordionSwift
